engine/admin/pruning.md

```diff
@@ -0,0 +1,187 @@
+objects at once. This topic shows how to use these `prune` commands.
+
+## Prune images
+
+The `docker image prune` command allows you to clean up unused images. By
+default, `docker image prune` only cleans up _dangling_ images. A dangling
+image is one that is not tagged and is not referenced by any running container.
+To remove dangling images:
+
+```bash
+$ docker image prune
+
+WARNING! This will remove all dangling images.
+Are you sure you want to continue? [y/N] y
+```
+
+To remove all images which are not used by existing containers, use the `-a`
+flag:
+
+```bash
+$ docker image prune -a
+
+WARNING! This will remove all images without at least one container associated to them.
+Are you sure you want to continue? [y/N] y
+```
+
+By default, you are prompted to continue. To bypass the prompt, use the `-f` or
+`--force` flag.
+
+You can limit which images are pruned using filtering expressions with the
+`--filter` flag. For example, to only consider images created more than 24
+hours ago:
+
+```bash
+$ docker image prune -a --filter "until=24h"
+```
+
+Other filtering expressions are available. See the
+[`docker image prune` reference](/engine/reference/commandline/image_prune.md)
+for more examples.
+
+## Prune containers
+
+When you stop a container, it is not automatically removed unless you started
+it with the `--rm` flag. To see all containers on the Docker host, including
+stopped containers, use `docker ps -a`. You may be surprised how many containers
+exist, especially on a development system! A stopped container's read-only and
+writable layers still take up disk space. To clean this up, you can use
+the `docker container prune` command.
+
+```bash
+$ docker container prune
+
+WARNING! This will remove all stopped containers.
+Are you sure you want to continue? [y/N] y
+```
+
+By default, you are prompted to continue. To bypass the prompt, use the `-f` or
+`--force` flag.
+
+By default, all stopped containers are removed. You can limit the scope using
+the `--filter` flag. For instance, the following command only removes
+stopped containers older than 24 hours:
+
+```bash
+$ docker container prune --filter "until=24h"
+```
+
+Other filtering expressions are available. See the
+[`docker container prune` reference](/engine/reference/commandline/container_prune.md)
+for more examples.
+
+## Prune volumes
+
+Volumes can be used by one or more containers, and take up space on the Docker
+host. Volumes are never removed automatically, because to do so could destroy
+data.
+
+```bash
+$ docker volume prune
+
+WARNING! This will remove all volumes not used by at least one container.
+Are you sure you want to continue? [y/N] y
+```
+
+By default, you are prompted to continue. To bypass the prompt, use the `-f` or
+`--force` flag.
+
+By default, all unused volumes are removed. You can limit the scope using
+the `--filter` flag. For instance, the following command only removes
+volumes older than 24 hours:
```

Comment:
volumes don't (yet) support the `until` filter, you could use an example using labels, e.g. 

prune all volumes that don't have the "keep" label;

```
docker volume prune --filter label!=keep
```